src/Main.java

```diff
@@ -1,10 +1,49 @@
-import java.lang.reflect.Array;
+
 import java.util.Arrays;
-import java.util.Random;
+
 
 public class Main {
     public static void main(String[] args) {
 
         System.out.println("Hello, World!");
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[5];
+
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000);
+        }
+
+        System.out.println(Arrays.toString(arr));
+
+        int total = 0;
+        for (int element : arr) {
+            total += element;
+        }
+        System.out.println(total);
+        int minInt = 1000000;
```

Comment:
Если минимальная значение станет 120000, то код перестанет работать. Рекомендую в таких алгоритмах приравнивать к значению первого элемента массив
```suggestion
        int minInt = arr[0];
```